nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    puts 'NotImplementedError'
+  end
```

Comment:
- [ ] Please make sure to raise the error and not to hard print it as requested in the project requirements.
  - You can consider this `raise NotImplementedError`.